app/multitenant/s3_client.go

```diff
@@ -84,6 +85,16 @@ func (store *S3Store) fetchReport(key string) (*report.Report, error) {
 	return report.MakeFromBinary(resp.Body)
 }
 
+// StoreReport serializes and stores a report.
+//
+// Returns the size of the report. This only equals bytes written if err is nil.
+func (store *S3Store) StoreReport(key string, report *report.Report) (int, error) {
+	var buf bytes.Buffer
+	report.WriteBinary(&buf, gzip.BestCompression)
+	err := store.StoreBytes(key, buf.Bytes())
+	return buf.Len(), err
+}
+
 // StoreBytes stores a report in S3, expecting the report to be serialized
 // already.
 func (store *S3Store) StoreBytes(key string, content []byte) error {
```

Comment:
Is this used somewhere other than StoreReport? Otherwise I would inline StoreBytes in StoreReport.